src/settings/ShowCapabilities.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import { stripesConnect, useOkapiKy } from '@folio/stripes/core';
+import { Button, Pane, Row, Col, SearchField } from '@folio/stripes/components';
+
+const ShowCapabilities = () => {
+  const ky = useOkapiKy();
+  const [capabilitiesResults, setCapabilitiesResults] = useState({});
+  const [capabilitySetsResults, setCapabilitySetsResults] = useState({});
+  const [query, setQuery] = useState('');
+
+  const searchParams = {
+    limit: 1000,
+    offset: 0,
+    query: `name=*${query}*`,
+  };
+
+  const submit = async () => {
+    const capabilitiesResponse = await ky.get('capabilities', { searchParams }).json();
+    setCapabilitiesResults(capabilitiesResponse);
+
+    const capabilitySetsResponse = await ky.get('capability-sets', { searchParams }).json();
+    setCapabilitySetsResults(capabilitySetsResponse);
+  };
+
+  const displayList = (resultList, listType) => {
+    return resultList[listType]?.map((value) => (
+      <ul>
+        <li>{value.permission}</li>
+        <ul>
+          <li><strong>type:</strong> {value.type}</li>
+          <li><strong>applicationId:</strong> {value.applicationId}</li>
+          <li><strong>resource:</strong> {value.resource}</li>
+          <li><strong>action:</strong> {value.action}</li>
+          <li><strong>permissionName:</strong> {value.name}</li>
+          <li><strong>permissionDisplayName:</strong> {value.description}</li>
+        </ul>
```

Comment:
`<ul>` is fine, but `* key: value` smells a lot like a `<dl>`.

Reply:
Seems like `stripes-components` `List` uses `<li>`. The formatting per the screenshot seems to work better with `<ul>` in this case.